task/source-build/0.1/source-build.yaml

```diff
@@ -0,0 +1,86 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: source-build
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "appstudio, hacbs"
+spec:
+  description: Source image build.
+  params:
+    - name: BINARY_IMAGE
+      description: Binary image name from which to generate the source image name.
+      type: string
+    - name: BASE_IMAGES
+      description: >-
+        Base images used to build the binary image. Each image per line in the same order of FROM
```

Comment:
> Base images used to build the binary image.

This could be a bit confusing (especially if read after previous parameter description) . Can we stress that it's Component's or project's binary image or something like this? But up to you.